qemu/tests/cfg/snp_multi_vm.cfg

```diff
@@ -0,0 +1,31 @@
+- snp_multi_vm:
+    type = snp_multi_vm
+    kill_vm = yes
+    login_timeout = 240
+    start_vm = no
+    image_snapshot = yes
+    vm_secure_guest_type = snp
+    vm_sev_reduced_phys_bits = 1
+    vm_sev_cbitpos = 51
+    virtio_dev_disable_legacy = on
+    bios_path =  /usr/share/edk2/ovmf/OVMF.amdsev.fd
+    snp_module_path = "/sys/module/kvm_amd/parameters/sev_snp"
+    module_status = Y y 1
+    snp_guest_check = "journalctl|grep -i -w snp"
+    guest_tool_install = "dnf install -y snpguest"
+    attestation_script = regular_attestation_workflow.sh
+    guest_dir = /home
+    guest_cmd = ${guest_dir}/${attestation_script}
+    host_script = sev-snp/${attestation_script}
+    variants:
+        - single_vcpu:
+            vms += " vm2"
+            smp_fixed = 1
+            vcpu_maxcpus=${smp_fixed}
```

Comment:
remove this line

Reply:
if don't set vcpu_maxcpus, the cpu cmdline will be
-smp 1,maxcpus=48,cores=1,threads=1,dies=1,sockets=1
[qemu output] qemu-kvm: Invalid CPU topology: product of the hierarchy must match maxcpus: sockets (1) * dies (1) * modules (1) * cores (1) * threads (1) != maxcpus (48)